templates/layout.html

```diff
@@ -38,6 +38,7 @@ <h1>{{ title }}</h1><hr>
 </div>
 {% endblock %}
 {% include 'footer.html' %}
+<script src="{{ url_for('static', filename='lib/js/commons.js') }}"></script>
```

Comment:
What's the magic here? `commons.js` as well as `lib.js` seem to contain the external dependencies and our own code is in `common.js` only, am I seeing this right?

Reply:
![image](https://user-images.githubusercontent.com/1881725/49731941-37ad0e80-fc7d-11e8-97d9-e810b3a489e0.png)
`yarn build:js:perf` and open `BundleReport.html` to see yourself

commons is whats shared between lib and numbers.